Create Nethereum metapackage

webwarrior-ws opened the pull request:
Create Nethereum metapackage and publish it on nuget.org.

knocte committed:
GitHubCI: add metapackage push

Following:
https://danielwertheim.se/how-to-create-a-nuget-metapackage/

Co-authored-by: Mehrshad <code.rezaei@gmail.com>

webwarrior-ws committed:
GithubCI: use nugetPreRelease.fsx to get version

webwarrior-ws committed:
GithubCI: publish package on nuget

knocte commented on .github/workflows/dotnet.yml:
```diff
@@ -22,3 +22,54 @@ jobs:
+          --property:TargetFrameworks=netstandard2.0 \
+          --property:Version=$VERSION
+    
+    - name: Push
```
@webwarrior-ws please add an `if` here that has three conditions:
1. checks secrets.NUGET_API_KEY is not empty
2. checks github.event_name == 'push'
3. checks github.ref == 'refs/heads/upstream'

webwarrior-ws replied:
Done

webwarrior-ws committed:
GithubCI: add conditions for "Push" step

Check for presence of secrets.NUGET_API_KEY is made in script
itself, because Github actions don't have access to secrets
context in if statements (see [1]).

[1] https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability
